app/controllers/course/assessment/submission/answer/comments_controller.rb

```diff
@@ -7,7 +7,23 @@ class Course::Assessment::Submission::Answer::CommentsController < \
 
   delegate :discussion_topic, to: :@answer
 
+  def create
+    @answer.class.transaction do
+      @post.title = @assessment.title
+      if super && @answer.save
```

Comment:
Got it - I realised I went to a different `PostsConcern`, where I didn't see the `create` function.